`fatal error: 'bridge/bridge.c' file not found` when using with `go mod vendor`

Hi! I'm having some trouble importing this package in a project that uses vendored dependencies, I'm seeing the following error when running `vet`/trying to build:

```
# go.riyazali.net/sqlite
vendor/go.riyazali.net/sqlite/bridge.go:3:11: fatal error: 'bridge/bridge.c' file not found
 #include "bridge/bridge.c"
          ^~~~~~~~~~~~~~~~~
1 error generated.
```

I _think_ it's related to this issue: https://github.com/golang/go/issues/26366 and I wonder if the contents of `bridge/` should be moved up into the root of this repo? (at least according to comments in that thread)

If there's an alternative solution I can do on my side, please let me know! Thanks so much

--- a/bridge/vendor_fix.go
+++ b/bridge/vendor_fix.go
@@ -1,0 +1,7 @@
+// +build dummy
+
+// Package bridge contains golang to c shim functions.
+//
+// This Go file is part of a workaround for `go mod vendor`.
+// Please see the file vendor_fix.go at the root of the module for more information.
+package bridge

--- a/h/vendor_fix.go
+++ b/h/vendor_fix.go
@@ -1,0 +1,7 @@
+// +build dummy
+
+// Package h contains C heaer files.
+//
+// This Go file is part of a workaround for `go mod vendor`.
+// Please see the file vendor_fix.go at the root of the module for more information.
+package h

--- a/vendor_fix.go
+++ b/vendor_fix.go
@@ -1,0 +1,11 @@
+// +build dummy
+
+// This file is part of a workaround for `go mod vendor` which won't vendor
+// C files if there's no Go file in the same directory.
+// This would prevent the bridge/bridge.c file to be vendored.
+//
+// See this issue for reference: https://github.com/golang/go/issues/26366
+package sqlite
+
+import _ "go.riyazali.net/sqlite/h"
+import _ "go.riyazali.net/sqlite/bridge"

